### ISSUE TITLE
Bitshuffle not in dials-1.2 binary package

### ISSUE THREAD

**Issue by graeme-winter** [dials/dials#133]
... or not registered

```
  File "h5py/_proxy.pyx", line 130, in h5py._proxy.dset_rw (/scratch/jenkins_slave/workspace/dials_release_build/compilationtarget/native/label/dials-ws133/build_dials/base_tmp/h5py-2.4.0/h5py/_proxy.c:1670)
  File "h5py/_proxy.pyx", line 84, in h5py._proxy.H5PY_H5Dread (/scratch/jenkins_slave/workspace/dials_release_build/compilationtarget/native/label/dials-ws133/build_dials/base_tmp/h5py-2.4.0/h5py/_proxy.c:1336)
IOError: Please report this error to dials-support@lists.sourceforge.net: Can't read data (Required filter 'bitshuffle; see https://github.com/kiyo-masui/bitshuffle' is not registered)
```

**Comment by graeme-winter** [comment]
This is based on installation last night from the 1.2 binaries from github.io on RHEL6

**Comment by graeme-winter** [comment]
Plugin appears to be present:


[gw56@cs04r-sc-com99-02 dials-v1-2-0]$ find . -name '*bitshuffle*'
./base/lib/python2.7/site-packages/bitshuffle-0.2.3.dev1-py2.7.egg-info
./base/lib/python2.7/site-packages/bitshuffle

**Comment by graeme-winter** [comment]
[gw56@cs04r-sc-com99-02 dials-v1-2-0]$ ls ./base/lib/python2.7/site-packages/bitshuffle
ext.so  h5.so  __init__.py  plugin  tests

**Comment by graeme-winter** [comment]
Possibly useful:


[gw56@cs04r-sc-com99-02 tests]$ dials.python test_h5filter.py
Traceback (most recent call last):
  File "test_h5filter.py", line 12, in <module>
    from bitshuffle import h5
  File "/dls/tmp/gw56/dials/dials-v1-2-0/base/lib/python2.7/site-packages/bitshuffle/__init__.py", line 19, in <module>
    from bitshuffle.ext import (__version__, bitshuffle, bitunshuffle, using_SSE2,
ImportError: /dls/tmp/gw56/dials/dials-v1-2-0/base/lib/python2.7/site-packages/bitshuffle/ext.so: undefined symbol: omp_set_lock

**Comment by graeme-winter** [comment]
OK, not plugin path stuff: 

[gw56@cs04r-sc-com99-02 tst]$ HDF5_PLUGIN_PATH=/dls/tmp/gw56/dials/dials-v1-2-0/base/lib dials.find_spots datablock.json nproc=20

=> same boom

**Comment by graeme-winter** [comment]
All plugins seem to be present & correct - therefore suspect that the omp link in the error above is the core of the problem; testing hypothesis by copying in my binaries of the libraries => works

**Comment by Anthchirp** [comment]
can you point me to a test dataset or datablock.json?

**Comment by graeme-winter** [comment]
/dls/mx-scratch/gw56/Eiger/NSLSII-16M/tst

**Comment by jmp1985** [comment]
Just a note, you're looking for the python bitshuffle module. You also need to check that these libraries are present (or some variant):

libh5bshuf.so  libh5zlz4.so    libh5zlz4.so.0.2
libh5LZF.so    libh5zlz4.so.0

**Comment by graeme-winter** [comment]
@jmp1985 yes, found that the particular libraries were compiled wrong; @Anthchirp has sorted I believe. I performed "dumb experiment" i.e. clobber the libraries in dials-1.2 binary with my own, started working after that. the .so.n.m stuff I think we can safely ignore...

**Comment by Anthchirp** [comment]
Fixed in 1.2.1 release https://github.com/dials/dials/releases/tag/v1.2.0
